revolv/static/css/screen.css

```diff
@@ -10677,7 +11056,122 @@ header.edit-project-header > .container > h1 {
   #modal-how-it-works .info-row .video-area .btn-play:hover .icons, #modal-how-it-works .info-row .video-area .btn-play:focus .icons {
     background-position: -48px -44px;
   }
+
+
+/* 30-Jun */
```

Comment:
Same thing with this comment. Please update it to describe what the styles below apply to.